tests/sklearn/test_sklearn_models.py

```diff
@@ -412,6 +434,7 @@ def check_serialization_dump_load(model, x, use_dump_method):
             serialized_model_dict["serialized_value"].pop(attribute, None)
             re_serialized_model_dict["serialized_value"].pop(attribute, None)
 
+        # Check if the graphs are similar
```

Comment:
this comment is wrong I think, we check if the serialized models are identical, not only the graphs